### ISSUE TITLE
Undefined initialStateTuple.a.state.a

### ISSUE THREAD

**Issue by petteraas** [hmsk/vite-plugin-elm#41]
When testing version 2.4.0 ( first found it in 2.4.1 , 2.3.1 appears to not have this issue ) I got a 

```
Uncaught TypeError: Cannot read property 'a' of undefined
``` 
that I've traced to this line :

https://github.com/hmsk/vite-plugin-elm/commit/9c910a2b8c92000afc13bcd4118bc5a07b53fed0#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R313

```
initializingInstance.initialState = JSON.stringify(initialStateTuple.a.state.a)
```
```
initialStateTuple.a
```
exists, but doesn't contain any `.state`

recreated both vith vite@2.3.3 and vite@2.4.1

**Comment by hmsk** [comment]
Thanks for the report! I will work on fixing it.
Will update this issue when the fix goes out.

**Comment by hmsk** [comment]
Fixed in v2.4.2 🚀
